suite-native/graph/src/components/Graph.tsx

```diff
@@ -68,11 +76,13 @@ export const Graph = ({ points = [], loading = false }: GraphProps) => {
                         style={applyStyle(graphStyle)}
                         points={graphPoints}
                         color={defaultColorVariant.green}
-                        animated
+                        animated={animated}
                         verticalPadding={20}
                         enablePanGesture
                         TopAxisLabel={axisLabels?.TopAxisLabel}
                         BottomAxisLabel={axisLabels?.BottomAxisLabel}
+                        onPointSelected={onPointSelected as any /* because of ExtendedGraphPoint */}
```

Comment:
Would it help if we typed this as `unknown`?

Reply:
It will throw. Once I will have time I will fork that graph library and do some changes to not need this.